dev/com.ibm.ws.webcontainer.servlet.4.0_fat.2/publish/servers/servlet40_webSphereServletEvent/server.xml

```diff
@@ -0,0 +1,29 @@
+<!--
+    Copyright (c) 2025 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+ -->
+<server description="Server for testing the websphere servlet event listener">
+
+    <include location="../fatTestCommon.xml"/>
```

Comment:
Why not use fatTestPorts.xml and use the httpEndpoint there rather than defining here in the server.xml?